client/blocks/eligibility-warnings/index.jsx

```diff
@@ -17,66 +20,90 @@ const EligibilityWarnings = props => {
 		translate,
 		backUrl,
 		isEligible,
-		errors,
-		pluginWarnings,
-		widgetWarnings,
+		eligibilityData
 	} = props;
 
+	// Mapping eligibility hold constant to messages that will be shown to the user
+	// TODO: update supportUrls and maybe create similar mapping for warnings
+	const holdsMessage = {
+		MULTIPLE_USERS: {
+			title: translate( 'Multiple users' ),
+			description: translate( 'This feature is not supported on sites with multiple users.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		NO_BUSINESS_PLAN: {
+			title: translate( 'Business plan required' ),
+			description: translate( 'This feature is only allowed on sites with business plan.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		NO_VIP_SITES: {
+			title: translate( 'VIP Site' ),
+			description: translate( 'This feature is not supported on VIP sites.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		NO_WPCOM_NAMESERVERS: {
+			title: translate( 'No WordPress.com Name Servers' ),
+			description: translate( 'Your custom domain must point to WordPress.com name servers.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		NON_ADMIN_USER: {
+			title: translate( 'Admin access required' ),
+			description: translate( 'Only site administrators are allowed to use this feature' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		NOT_USING_CUSTOM_DOMAIN: {
+			title: translate( 'No custom domain' ),
+			description: translate( 'Your site must use custom domain in order for this feature to work.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		SITE_GRAYLISTED: {
+			title: translate( 'Flagged site' ),
+			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		SITE_PRIVATE: {
+			title: translate( 'Private site' ),
+			description: translate( 'This feature is not supported on private sites.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		}
+	};
```

Comment:
This is fine, but feel free to move this into a function in the module scope which accepts `translate` as an argument. That can clean up the component proper so less noise is in the render path.